tests/pass/intrinsics-x86-sse2.rs

```diff
@@ -0,0 +1,742 @@
+//@only-target-x86_64
```

Comment:
I now realize this also applies to your other test file, but -- shouldn't we also test those intrinsics on 32bit x86?

Reply:
Is there a directive to specify more than one target?

Reply:
Hm, multiple "only" are interpreted conjunctively so that won't work.

But its' a substring test, so `only-target-x86` should work.

Reply:
Does the directive check the target triple (x86_64-unknown-linux-gnu, i686-unknown-linux-gnu) or the target arch name (x86_64, x86)?

Reply:
It's a substring match on the entire triple.